packages/eslint-plugin/docs/rules/no-unnecessary-type-parameters.md

```diff
@@ -0,0 +1,104 @@
+---
+description: 'Disallow type parameters that only appear once'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-unnecessary-type-parameters** for documentation.
+
+This rule forbids type parameters that only appear once in a function, method, or class definition.
+
+Type parameters relate two types. If a generic type parameter only appears once, then it is not relating anything. It can usually be replaced with explicit types such as `unknown`.
```

Comment:
[Docs] Nit: "generic" in "generic type parameter" is redundant, I think. As, all type parameters are "generic".

```suggestion
Type parameters relate two types. If a type parameter only appears once, then it is not relating anything. It can usually be replaced with explicit types such as `unknown`.
```